src/style.css

```
@@ -0,0 +1,84 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.wrap {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.blocks {
+  display: flex;
+  color: #fff;
+  height: 300px;
+  min-width: 300px;
+  flex-grow: 1;
+  flex-basis: 33.33%;
+  justify-content: center;
+  align-items: center;
+}
+
+.block--1 {
+  order: 1;
+  background-color: rgb(255 0 0);
+}
+
+.block--2 {
+  order: 2;
+  background-color: rgb(204 0 0);
+
+}
+
+.block--3 {
+  order: 3;
+  background-color: rgb(153 0 0);
+}
+
+.block--4 {
+  order: 4;
+  background-color: rgb(102 0 0);
+}
+
+.block--5 {
+  order: 5;
+  background-color: rgb(51 0 0);
+}
+
+.block--6 {
+  order: 6;
+  background-color: rgb(0 0 0);
+}
+
+@media (min-width: 600px) and (max-width: 900px) {
```

It's not needed here, delete this
```suggestion
@media (min-width: 600px) {
```